src/components/Accordion/Accordion.test.tsx

```diff
@@ -160,19 +160,17 @@ describe("Accordion", () => {
   });
 
   it("closes the accordion when the 'esc' key is pressed", async () => {
-    render(<Accordion accordionData={[accordionData[0]]} isDefaultOpen />);
+    render(<Accordion accordionData={[accordionData[0]]} />);
```

Comment:
Can you add another test where the entire `accordionData` array is passed, so now there are three accordions. For one of the accordions, add a button or checkbox, focus on it, and then press escape. This is to test the new feature in this PR but also that it works when there are more than just one accordion.

Reply:
I know there's a test that does this for `MultiSelect` but the feature is in the Accordion so it should also be tested there.

Reply:
Done!